test/robot/stash/factory.go

```diff
@@ -49,6 +50,16 @@ func Dial(ctx context.Context, address string) (*Client, error) {
 			location.Scheme = "file"
 		}
 	}
+	if location.Scheme == "file" {
+		if runtime.GOOS == "windows" {
+			if l := len(location.Path); l > 2 && location.Path[0] == '/' && location.Path[2] == ':' {
+				// windows file urls have an extra slash before the volume label that needs to be remove
+				// see https://github.com/golang/go/commit/844b625ebcc7101e09fb87828a0e71db942a2416d
```

Comment:
accidentally moved the 'd' from "removed" on the previous line, which broke the link.